### ISSUE TITLE
The contenteditable attribute selector is not correct

### ISSUE THREAD

**Issue by dserv-nh** [st-h/ember-content-editable#105]
Hello,

In [this pull request](https://github.com/st-h/ember-content-editable/pull/84) you added 
` -webkit-user-modify: read-write-plaintext-only;`
to the `[contenteditable]` selector.

I think this is not correct because it may have a `"false"` value. We use TinyMCE which set's this property to false when you want the content to be read only and also [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable) there is a reference to the `"false"` option.
I think you should adjust the selector to only select elements that are truly editable.

**Comment by st-h** [comment]
@dserv-nh thanks for opening this issue. If I understand you correctly you are suggesting to change the selector to?

```
[contenteditable="true"] {
   cursor: text;
   -webkit-user-modify: read-write-plaintext-only;
 }
```

**Comment by dserv-nh** [comment]
Yes, but I you need to be careful because your component only has `contenteditable` without an argument. Therefore the `[contenteditable="true"]` would not work on this.
From MDN:
> If the attribute is given without a value, like `<label contenteditable>Example Label</label>`, its value is treated as an empty string.

But I think you should consider using a more specific selector to the ember-content-editable component and not changing this globally for every element with `contenteditable` so something like `.ember-content-editable[contenteditable=""]` should be more appropriate

**Comment by dserv-nh** [comment]
To solve this issue, can you merge [this pull request](https://github.com/st-h/ember-content-editable/pull/99)?

**Comment by st-h** [comment]
sorry, have been very busy with a lot of other stuff. There have been a few minor things I wanted to fix/upgrade as well. Try to get it done asap. Just merged the PR.

**Comment by st-h** [comment]
sorry about the late reply. This should be now fixed in version 3.0.0. Upgrades require to migrate due to two way binding no longer being available with current ember versions.
